source/Patches/EndGameSummary.cs

```diff
@@ -2,23 +2,29 @@
 using System.Collections.Generic;
 using System.Text;
 using UnityEngine;
+using TownOfUs.Extensions;
 using TownOfUs.Roles;
 using TownOfUs.Roles.Modifiers;
 using HarmonyLib;
+using System.Linq;
 
 namespace TownOfUs.Patches
 {
     [HarmonyPatch]
     static class EndGameSummary
     {
-        static class AdditionalTempData
+        public static class AdditionalTempData
```

Comment:
It's not really temporary data anymore since it's being hung onto all game. Might want to call it something like `RoleHistory`?

Reply:
to be fair, "AdditionalTempData" is a name that i borrowed from TOR, I assumed they used the name to mean "temporary for that game", and as the data is wiped at the end of the game, i assumed it was still temporary, when i get the time i will look into changing it to something more appropriate.

Reply:
now Renamed to "AdditionalGameData"